zhaquirks/xiaomi/aqara/opple_switch.py

```diff
@@ -52,28 +53,28 @@
 PRESS_TYPES = {0: "hold", 1: "single", 2: "double", 3: "triple", 255: "release"}
 
 
-class OppleOperationMode(t.uint8_t):
+class OppleOperationMode(t.uint8_t, Enum):
```

Comment:
Unfortunately not. I then get data type errors when I set the Attribute

Reply:
This is the error I got:
`[homeassistant.components.zha.api] Set attribute for: cluster_id: [64704] cluster_type: [in] endpoint_id: [1] attribute: [512] value: [False] manufacturer: [4447] response: [Write_Attributes_rsp(status_records=[WriteAttributesStatusRecord(status=<Status.INVALID_DATA_TYPE: 141>, attrid=0x0200)])]`

Reply:
Are you maybe trying to set the values as `True`/`False` instead of 1/0?

It had been implemented this way in other places:
https://github.com/zigpy/zha-device-handlers/blob/dev/zhaquirks/tuya/__init__.py#L849..L853

But it is known that it does not work with boolean values.

In any case if it is working properly I don't see any problem with keeping it as is.

Reply:
In the UI? I manually entered 1/0 but also the configuration entity failed to set the values with the same error (no UI feedback, only in the log?